elements/rh-band/demo/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>RHElements: rh-band Demo</title>
+
+    <noscript>
+      <link href="../../rhelement/rhelement-noscript.min.css" rel="stylesheet">
+    </noscript>
+
+    <link href="../../rhelement/rhelement.min.css" rel="stylesheet">
+
+    <!-- uncomment the es5-adapter if you're using the umd version -->
+    <script src="/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
+    <script src="/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.5/require.min.js"></script>
+    <script>require([
+      '../rh-band.umd.js',
+      '../../rh-card/rh-card.umd.js',
+      '../../rh-cta/rh-cta.umd.js'
```

Comment:
#nice-to-have  we could add
`      '../../rh-layouts/rh-layouts.umd.js'`
here, and show one demo using those classes on the content container